### DISCUSSION TITLE
404 Not Found

### DISCUSSION THREAD

**Discussion by MargulanDev** [malikzh/NCANode#148] [Q&A]
Получаю ошибку на все запросы:
{
    "timestamp": "2022-12-01T09:38:11.335+00:00",
    "status": 404,
    "error": "Not Found",
    "path": "/"
}

В чем может быть ошибка? Есть ли актуальный postman json?

**Comment by MargulanDev** [comment]
Вся проблемы была из-за того что в документациях нет роутов. Пришлось разобрать код чтобы оттуда роуты вытащить

**Comment by malikzh** [comment]
Вот тут находится документация: https://v3.ncanode.kz/swagger-ui/
А также в репе есть openapi.yml в котором все описано.

чт, 1 дек. 2022 г. в 17:33, MargulanDev ***@***.***>:

> Вся проблемы была из-за того что в документациях нет роутов. Пришлось
> разобрать код чтобы оттуда роуты вытащить
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/malikzh/NCANode/discussions/148#discussioncomment-4282579>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIKJ3TKRZP4XENFMIQWIIKDWLCEJ7ANCNFSM6AAAAAASQSKR4U>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
С уважением, Malik Zharykov.
*Software Engineer*
E-mail: ***@***.***
GitHub: https://github.com/malikzh
Twitter: https://twitter.com/malikz_h
LinkedIn: https://www.linkedin.com/in/malik-zharykov-bba687165/
